src/proteus/dswx_hls.py

```diff
@@ -178,55 +186,25 @@
 2. Partial surface water.
 '''
 collapse_wtr_classes_dict = {
-    WTR_NOT_WATER: WTR_NOT_WATER,
-    WTR_UNCOLLAPSED_HIGH_CONF_WATER: WTR_COLLAPSED_OPEN_WATER,
-    WTR_UNCOLLAPSED_MODERATE_CONF_WATER: WTR_COLLAPSED_OPEN_WATER,
-    WTR_UNCOLLAPSED_PARTIAL_SURFACE_WATER_CONSERVATIVE: \
-        WTR_COLLAPSED_PARTIAL_SURFACE_WATER,
-    WTR_UNCOLLAPSED_PARTIAL_SURFACE_WATER_AGGRESSIVE: \
-        WTR_COLLAPSED_PARTIAL_SURFACE_WATER,
+    WATER_NOT_WATER_CLEAR: WATER_NOT_WATER_CLEAR,
+    WATER_UNCOLLAPSED_HIGH_CONF_CLEAR: WATER_COLLAPSED_OPEN_WATER,
+    WATER_UNCOLLAPSED_MODERATE_CONF_CLEAR: WATER_COLLAPSED_OPEN_WATER,
+    WATER_UNCOLLAPSED_PARTIAL_SURFACE_WATER_CONSERVATIVE_CLEAR: \
+        WATER_COLLAPSED_PARTIAL_SURFACE_WATER,
+    WATER_UNCOLLAPSED_PARTIAL_SURFACE_WATER_AGGRESSIVE_CLEAR: \
+        WATER_COLLAPSED_PARTIAL_SURFACE_WATER,
     WTR_OCEAN_MASKED: WTR_OCEAN_MASKED,
     WTR_SNOW_MASKED: WTR_SNOW_MASKED,
     WTR_CLOUD_MASKED: WTR_CLOUD_MASKED,
     UINT8_FILL_VALUE: UINT8_FILL_VALUE
 }
 
-'''
-Dictionary containing the mapping from the output 2-water classes
-(after collapsing) to confidence values in percent
-'''
-wtr_confidence_dict = {
-    WTR_NOT_WATER: CONF_NOT_WATER,
-    WTR_COLLAPSED_OPEN_WATER: 85,
-    WTR_COLLAPSED_PARTIAL_SURFACE_WATER: 70,
-    WTR_OCEAN_MASKED: CONF_OCEAN_MASKED,
-    WTR_SNOW_MASKED: CONF_SNOW_MASKED,
-    WTR_CLOUD_MASKED: CONF_CLOUD_MASKED,
-    UINT8_FILL_VALUE: UINT8_FILL_VALUE
-}
-
-'''
-Dictionary containing the mapping from the original 4-water classes
-(before collapsing) to confidence values in percent
-'''
-wtr_confidence_non_collapsed_dict = {
-    WTR_NOT_WATER: CONF_NOT_WATER,
-    WTR_UNCOLLAPSED_HIGH_CONF_WATER: 95,
-    WTR_UNCOLLAPSED_MODERATE_CONF_WATER: 80,
-    WTR_UNCOLLAPSED_PARTIAL_SURFACE_WATER_CONSERVATIVE: 70,
-    WTR_UNCOLLAPSED_PARTIAL_SURFACE_WATER_AGGRESSIVE: 60,
-    WTR_OCEAN_MASKED: CONF_OCEAN_MASKED,
-    WTR_SNOW_MASKED: CONF_SNOW_MASKED,
-    WTR_CLOUD_MASKED: CONF_CLOUD_MASKED,
-    UINT8_FILL_VALUE: UINT8_FILL_VALUE
-}
-
 collapsable_layers_list = ['WTR', 'WTR-1', 'WTR-2']
 
 band_description_dict = {
     'WTR': 'Water classification (WTR)',
     'BWTR': 'Binary Water (BWTR)',
-    'CONF': 'TBD Confidence (CONF)',
+    'CONF': 'Confidence classifications (CONF)',
```

Comment:
Classifications or classification? Maybe we can just call it `Confidence layer`

Reply:
I kinda like calling it "classification" (singular), to help people remember that this values do not reflect percentages. I'll update to "classification" now, but if you feel strongly about changing it in your PR after this, "Confidence layer" makes sense too.